src/content/admin/resource-manager.mdx

```diff
@@ -0,0 +1,127 @@
+It calculates the average usage and automatically adjusts the resource requests for new pods based on these averages. The system uses the following process:
+
+1. Data Collection: The Resource Manager queries the Kubernetes Metric Server to collect CPU and memory usage data for all pods managed by Okteto
+1. Average Calculation: It calculates the average CPU and memory usage for each container family (grouped by development environment, component, and container name)
+1. Recommendation Generation: The Resource Manager generates new resource requests based on the average usage, applying a correction factor to ensure efficiency
+1. Automatic Application: When a new pod is created, the recommended resource requests are automatically applied unless overridden by specific annotations
+
+## Enabling and Configuring the Resource Manager
+### Enabling the Resource Manager
+The Resource Manager is controlled through the Helm chart configuration. By default, it is enabled with the following settings:
+
+``` yaml
+resourceManager:
+  enabled: true
+  schedule: 300
+  deletePeriod: 15
+  priorityClassName:
+    labels:
+	annotations:
+	resources:
+	recommendations:
+		weight:
+		correction:
+		min:
+			cpu:
+			memory:
+```
+To enable the Resource Manager:
+
+1. Update your Helm values file to include the `resourceManager` block from above
+1. Deploy your Helm chart with the updated configuration
+1. The Resource Manager will start running as a cron job every 300 seconds (5 minutes)
+
+### Enabling via the Okteto Admin UI
+If the Resource Manager is not already enabled via Helm, or if you prefer to manage it from the UI, follow these steps:
+
+1. Log in to the Okteto Admin UI
+1. Navigate to Admin -> General where you can manage system-level settings
+1. Locate the Resource Manager section. If the Resource Manager is installed but not yet enabled, you will see an "Enable/Disable" toggle switch
+1. Toggle the switch to "Enable". This will activate the Resource Manager without needing to manually update the Helm chart
+
+[Add Screenshot here]
+
+Once enabled, the Resource Manager will immediately begin monitoring and adjusting resource requests based on your specified schedule.
+
+### Disabling the Resource Manager
+If you need to disable the Resource Manager temporarily, you can do so either by:
+
+- Using the toggle switch in the Admin UI to dynamically disable the feature without redeploying the chart
+- Removing or setting `enabled: false` in the Helm values file and redeploying your Helm chart
+
+### Customizing Resource Manager Settings
+You can customize the Resource Manager by adjusting the following settings in your Helm values file:
+
+- `schedule`: Frequency (in seconds) at which the Resource Manager runs
+- `deletePeriod`: Number of days to retain resource data before deletion
+- `priorityClassName`: Assigns a priority class to the Resource Manager’s pods
+- `resources`: Defines resource requests and limits for the Resource Manager itself
```

Comment:
Are there more settings that we should add here?

Reply:
for reference, I've created a dedicated PR for the helm config: https://github.com/okteto/docs/pull/834 (still WIP)